panther_driver/src/velocity_smoother.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python3
+
+from geometry_msgs.msg import Twist
+
+
+class VelocitySmoother:
+    def __init__(
+        self,
+        max_vel_x,
+        max_vel_y,
+        max_vel_theta,
+        acc_lim_x,
+        acc_lim_y,
+        acc_lim_theta,
+        decel_lim_x,
+        decel_lim_y,
+        decel_lim_theta,
+        emergency_decel_lim_x,
+        emergency_decel_lim_y,
+        emergency_decel_lim_theta,
+    ) -> None:
+
+        self._max_vel_x = max_vel_x
+        self._max_vel_y = max_vel_y
+        self._max_vel_theta = max_vel_theta
+        self._acc_lim_x = acc_lim_x
+        self._acc_lim_y = acc_lim_y
+        self._acc_lim_theta = acc_lim_theta
+        self._decel_lim_x = decel_lim_x
+        self._decel_lim_y = decel_lim_y
+        self._decel_lim_theta = decel_lim_theta
+        self._emergency_decel_lim_x = emergency_decel_lim_x
+        self._emergency_decel_lim_y = emergency_decel_lim_y
+        self._emergency_decel_lim_theta = emergency_decel_lim_theta
+
+        self._robot_velocity = [0.0, 0.0, 0.0]  # vel_x, vel_y, vel_theta
```

Comment:
maybe instead of array we can just use 3 variables: _robot_velocity_x, _robot_velocity_y, _robot_velocity_theta